File: diploma.controller/src/kz/greetgo/diploma/controller/model/Blog.java
package kz.greetgo.diploma.controller.model;

public class Blog {
  
  public String id;
  public String title;
  public String article;
  public String imageId;


  @Override
  public String toString() {
    return "Blog{" +
      "id='" + id + '\'' +
      ", title='" + title + '\'' +
      ", article='" + article + '\'' +
      ", imagesId='" + imageId + '\'' +
      '}';
  }
}

File: diploma.client/src/model/Blog.ts
import {environment} from "../environments/environment";

export class Blog {
  public id: string;
  public title: string;
  public article: string;
  public imageId: string;
  public files: FileList;


  url() {

    if (this.imageId) {
      return environment.urlPrefix + '/files/get2?fileId=' + this.imageId;
    }
    
    return 'assets/img/blog1.jpg';
  }

}
